src/referrals/dto/create-referral-entity.dto.ts

```
@@ -0,0 +1,69 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
```

ESLint

[error] Replace `'@nestjs/swagger'` with `"@nestjs/swagger"`

(prettier/prettier)